resources/views/js-modal.blade.php

```diff
@@ -31,20 +32,20 @@ class="flex overflow-y-auto fixed inset-0 z-50 md:py-10 md:px-8"
 
     <div
         class="modal-content-wrapper md:m-auto w-full {{ $class }} {{ $widthClass }}"
-        @if(!$closeButtonOnly)
```

Comment:
this is a breaking change. Can it not make use of the `closeButtonOnly` in your scenario instead of needing an additional property to handle it? 🤔

Reply:
@ItsANameToo I've added because 'closeButtonOnly' also remove the top-right close button.
But you're right, this is a BC. In my opinion we can use the modal without the additional X button. In this case we don't need this PR anymore. What do you thing?